include/dbclean.php

```diff
@@ -21,6 +22,10 @@ function dbclean_run(&$argv, &$argc) {
 	load_config('config');
 	load_config('system');
 
+	if (!get_config("system", "dbclean")) {
```

Comment:
@Quix0r this is not really a code-converntion we try to group functions into classes (e.g. we have done this with the Config class + some other - more will come). `get_config()` is the legacy call because we don't have moved every call of `get_config()` to `Config::get()` (have a look at https://github.com/friendica/friendica/blob/develop/include/config.php#L45)

> And how about more wrappers, like Config::isset($group, $key)

`Config::get()` behaves this way. Have a look at https://github.com/friendica/friendica/blob/develop/include/Core/Config.php#L76 . If it isn't set it returns `null`